Daramola98/WEconnectApp
# Implements feature delete a  business

What does this PR do?
This PR enables users to successfully delete a business he/she registered.

#### Description of Task to be completed?

- Configure action creator to delete a business
- Set up confirm business deletion modal

### How should this be manually tested?
View application at weconnect-api.herokuapp.com
Login, then register a business if you have none
View your businesses on your user profile page 
To delete a business click on the delete button and confirm you want to delete the business

#### What are the relevant pivotal tracker stories?
#156799221
